redaxo/src/core/fragments/core/toolbar.php

```diff
@@ -4,7 +4,7 @@
 
 ?>
 
-<nav class="navbar navbar-<?= (isset($this->inverse) && $this->inverse) ? 'inverse' : 'default'; ?><?= (isset($this->cssClass) && $this->cssClass != '') ? ' ' . $this->cssClass : ''; ?>">
+<nav class="navbar navbar-<?= (isset($this->inverse) && $this->inverse) ? 'inverse' : 'default'; ?><?= (!isset($this->brand) || !$this->brand || $this->brand === '') ? ' rex-navbar-without-brand' : ' rex-navbar-without-brand' ?><?= (isset($this->cssClass) && $this->cssClass != '') ? ' ' . $this->cssClass : ''; ?>">
```

Comment:
könnte man statt `!isset($this->brand) || !$this->brand || $this->brand === ''` vllt `empty($this->brand)` verwenden?